### PR TITLE
Add Pkg 1.8 news

### PR DESCRIPTION
**Pull request by IanButterworth**

### CODE REVIEW

**Comment by IanButterworth** [comment]
It's not clear to me what the error is. There's a warning but no error apart from 
```
ERROR: LoadError: `makedocs` encountered an error. Terminating build
Stacktrace:
 [1] error(s::String)
   @ Base ./error.jl:35
```

**Comment by fredrikekre** [comment]
> But not sure how that is related to this PR.

This PR adds a new markdown header called `Pkg`

**Comment by IanButterworth** [comment]
I wonder if having doc refs in the form "Pkg#123" is treated as magic such that it ignores the # onwards

File: NEWS.md [hunk 1]
```diff
@@ -111,6 +111,26 @@ Standard library changes
 * Intersect returns a result with the eltype of the type-promoted eltypes of the two inputs ([#41769]).
 * Iterating an `Iterators.Reverse` now falls back on reversing the eachindex iterator, if possible ([#43110]).
 
+#### Pkg
```
Comment (IanButterworth) [inline NEWS.md:114]:
```suggestion
#### Package Manager
```
This is what the 1.7 news file does

**Pull request merged**